src/client/java/daos-java-api/src/main/java/com/intel/daos/client/DaosObjectType.java

```diff
@@ -0,0 +1,425 @@
+/*
+ * (C) Copyright 2018-2019 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. B609815.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+
+package com.intel.daos.client;
+
+/**
+ * Type of DAOS object.
+ */
+public enum DaosObjectType {
```

Comment:
this is really bad.. do you have to replicate the values?
again if we change any value in DAOS, we have to remember to change it hear, and most of the time this will not happen.

Reply:
Again, we don't want to narrow down native  DFS APIs, Thus, we have this class with values copied. Maybe we should use object type name which maps to DAOS value. Do you have better solution for this?